L1Trigger/L1TCalorimeter/python/caloParams_cfi.py

```diff
@@ -142,7 +142,7 @@
     minimumBiasThresholds = cms.vint32(0, 0, 0, 0),
     centralityLUTFile = cms.FileInPath("L1Trigger/L1TCalorimeter/data/centralityLUT_stage1.txt"),
     q2LUTFile         = cms.FileInPath("L1Trigger/L1TCalorimeter/data/q2LUT_stage1.txt"),
-    zdcLUTFile        = cms.FileInPath("L1Trigger/L1TZDC/data/zdcLUT_HI_v0_1.txt"),
+#    zdcLUTFile        = cms.FileInPath("L1Trigger/L1TZDC/data/zdcLUT_HI_v0_1.txt"),
```

Comment:
No strong reason beyond possible historical utility - if requested will delete as well.

Reply:
Hi @perrotta I can remove this commented line if  this is needed to continue the progress on this PR - let me know (and if there are any other requested changes at this time, happy to apply fixes as necessary)

Reply:
> Hi @perrotta I can remove this commented line if this is needed to continue the progress on this PR - let me know (and if there are any other requested changes at this time, happy to apply fixes as necessary)

I would personally remove it, but I am not expected to sign this PR: I think that it is up to @cms-sw/l1-l2 to decide whether any change is required

Reply:
@cfmcginn Catching up on things now after the holidays. I would remove it since this was temporary, and present in the history if truly necessary. I would value the clarity of the file over trying to preserve history by hand instead of letting git do it.

Reply:
Removed in latest commit - marking as resolved.